main/server/server.js

```diff
@@ -844,6 +848,22 @@ const setupListeners = (port, userDataPath) => {
               () => 'Error while getting current backups'
             )
           }
+          case LAST_OPENED_FILE: {
+            return handlePromise(
+              () => 'Getting the last opened file',
+              () => lastOpenedFile(),
```

Comment:
This task might take time, we should register it with the `statusManager`.
(Tasks registered with the `statusManager` will inform the renderer that the socket server is busy and also prevent Plottr from quitting.)